### ISSUE TITLE
[1.16.5] "Jerking Arrows"

### ISSUE THREAD

**Issue by Shibva** [Yesssssman/epicfightmod#861]
durring recent plays I have been bugged by this strange visual glitch when an arrow strikes me when im in my nomral model (mineing mod, config for this being shown when not in combat mode is set accordingly). what happens can be hard to explain, but think of how arrows lag sometimes and they reposing themselves as they make their way, now imagine just that happening only when the arrow impales you and is rendering on you but with no lag but it jerking and twiching like it. Ive been bugged by this and it was as of today that I decided to get to the bottom of this. 

by using an old binary, and some elimination, I found that this mod seems to be causing it. I will note that in the first test to make sure that it dint exist (and thank god it didnt) with this unusual behavior, but when I updated epic fight from 16.5.10 to the latest (16.5.13 as of posting this) that the issue became existent for this 

##TLDR##
a change made in between 16.5.11 to 16.5.13 resulted in this glitch happening to the normal player model when an arrow hits them, please investigate, ill be willing to help out in anyway I can
